src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costCarDay = 40;
+  const discountOverThreeDays = 20;
+  const discountOverSevenDays = 50;
+
+  if (days >= 7) {
+    return (days * costCarDay) - discountOverSevenDays;
```

Comment:
days * costCarDay create a variable for this calculations, you repeat this code three times